server/resources/resource_management.go

```diff
@@ -316,6 +316,23 @@ func GetMultipleResources(context middleware.Context, dataStore db.DB, resourceS
 	currCond := policy.GetCurrentResourceCondition()
 
 	filter := FilterFromQueryParameter(resourceSchema, queryParameters)
+	if search_filter,ok := queryParameters["search_field"]; ok && resourceSchema.IsSubstringSeachEnabled(){
```

Comment:
If we can enable substring searching on demand now, do we really need to have that also enabled in schemas?

Reply:
earlier this flag was disable_substring_search...earlier reviews suggested no to stick to negative logic ..so changed it to enable_substring_search..probably the enable might be confusing..will try to find a different name
https://github.com/cloudwan/gohan/pull/731#discussion_r262837149